vignettes/articles/cookbook_interpretation.Rmd

```diff
@@ -80,19 +91,17 @@ qplot_techmix(
+The production volume trajectory metric evaluates a portfolio’s alignment with climate scenarios by comparing projected production volumes---based on companies' five-year capital expenditure plans---to scenario targets. It applies to the fossil fuels, power, and automotive sectors, where production changes occur either through shifts between technologies (e.g., internal combustion engines to electric vehicles) or overall expansion/contraction (e.g., adding new coal-fired power plants). The trajectory reflects these changes at the technology level for individual clients and is compared to climate scenario trends.
 
 #### Explanation of Output
 
-The production volume trajectory plot shows the production volume of the portfolio or company for one technology (or fuel type) at a time. The plot shows relative changes over a time frame between the start year of the analysis and five years into the future (solid black line). This projected production volume of the portfolio is compared to the production volume in the benchmark region over the same period of time (dashed black line). Finally the background of the plot is shaded in a color scale from red to yellow and green. The lines between the shaded areas represent the relative changes for the technology based on the scenario, using the market share approach. For low-carbon technologies, anything above the most ambitious scenario will be displayed in green, anything below the least ambitious scenario will be displayed in red. For high-carbon technologies, anything below the most ambitious scenario will be displayed in green, anything above the least ambitious scenario will be displayed in red. Generally, the production volume trajectory plot shows multiple scenarios from the same scenario publication, although it also works when comparing the portfolio to a single scenario (the scenario line would split the background in two shaded areas in that case).
+The trajectory plot displays a portfolio’s production volume for a specific technology or fuel type, tracking changes from the start year to five years into the future (**solid black line**). This projection is compared to the benchmark region’s production trajectory (**dashed black line**). The plot’s background is color-coded: green indicates alignment with ambitious climate scenarios, red signifies misalignment, and shades of yellow for in-between. For low-carbon technologies, values above the most ambitious scenario appear green, and those below the least ambitious scenario appear red; the opposite applies to high-carbon technologies. Typically, the plot includes multiple scenarios but can also compare the portfolio against a single scenario.
```

Comment:
Since this plot is a frequent subject of confused questions, I thought it may be better to clarify that scenario pathways are the lines delineating the colors. I am also realizing that we wanted to clarify what alignment in this plot means in terms of temperature outcome (another recurring question). I don't think we do that at the moment.